build-tools/src/main/resources/intellij/ZeebeStyle.xml

```diff
@@ -115,54 +118,6 @@
         </group>
       </groups>
       <rules>
-        <section>
-          <rule>
-            <match>
-              <AND>
-                <FIELD>true</FIELD>
-                <FINAL>true</FINAL>
-                <PUBLIC>true</PUBLIC>
-                <STATIC>true</STATIC>
-              </AND>
-            </match>
-          </rule>
-        </section>
-        <section>
-          <rule>
-            <match>
-              <AND>
-                <FIELD>true</FIELD>
-                <FINAL>true</FINAL>
-                <PROTECTED>true</PROTECTED>
-                <STATIC>true</STATIC>
-              </AND>
-            </match>
-          </rule>
-        </section>
-        <section>
```

Comment:
Why are these removed?

Reply:
I removed the rules that enforced grouping final fields (static and instance) together, since the check would flag them as wrong (since it'd order final private above non-final public, for example).

Reply:
ok thanks